actor/builtin/miner/miner_test.go

```diff
@@ -1254,41 +1256,41 @@ func TestVerifyPIP(t *testing.T) {
 		require.Nil(t, verifier.LastReceivedVerifyPieceInclusionProofRequest, "should have never called the verifier")
 	})
 
-	t.Run("PIP is invalid if miner has not submitted any PoSt proofs", func(t *testing.T) {
-		vmctx, verifier, minerActor := (&minerEnvBuilder{
-			message:   "verifyPieceInclusionProof",
-			sectorSet: sectorSetWithOneCommitment,
-			lastPoSt:  nil,
-		}).build()
+	t.Run("PIP is invalid if miner is tardy/slashable", func(t *testing.T) {
```

Comment:
Non-Blocking: It would be nice to have an additional test for the condition where the miner is not active.